### DISCUSSION TITLE
Add Page Index Link option?

### DISCUSSION THREAD

**Discussion by punksinatra** [GetPublii/Publii#1632] [Ideas]
![image](https://github.com/user-attachments/assets/f5609c03-42db-4bb8-90b8-25e7fc2bf5da)

Is it possible to add option in the menu for type "Page index link"? 

I want to display/show some pages (subpages) on a main page but I can't seem to figure out how to do so. Since pages does not support (tagging) I cannot use "tags list link" as well.

Or is there any way to add a parent page to the menu and when you click on it goes to the parent page and it will show subpages under that parent page?

**Comment by bobmitro** [comment]
Pages are separate from the loop and serve as standalone content. Otherwise, there would be no point in having posts and pages if they work the same way. 
So, there is no such thing as a page index.

 If you want to display a list of pages, you can use helpers like [getpages](https://getpublii.com/dev/getpages-helper/)..."

**Reply by punksinatra** [reply]
Thank you for pointing me to getpages
